Gemfile.lock

```diff
@@ -1,17 +1,18 @@
 PATH
   remote: .
   specs:
-    time_range_extractor (0.2.3)
+    time_range_extractor (0.3.5)
       activesupport (> 2.0)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (5.2.3)
+    activesupport (6.0.3)
```

Comment:
can you lock your ruby version here - we don't want to have a requirement for rails 6.

Reply:
How should the ruby version be locked?

The gemfile is generated from the gemspec and the gemspec has a requirement of `activesupport > 2`.  I don't think it makes sense to lock it to `< 6`.

Reply:
You can lock it to in the gemspec. we should do that for now to make sure it's not breaking it.

Reply:
I downgraded activesupport.

I'm not clear what the concern was?

Reply:
not 100% but when we update the main app to use this gem, i was worried that if we're tied to rails6+ then it'd create dependency conflict